src/app.module.ts

```
@@ -1,11 +1,41 @@
+		}),
+		TripsModule,
+	],
 	controllers: [AppController],
 	providers: [AppService],
 })
-export class AppModule {}
+export class AppModule implements OnApplicationShutdown {
+	private static mongoMemoryServer: MongoMemoryServer
+
+	static async getMongoMemoryServer(): Promise<MongoMemoryServer> {
+		if (!this.mongoMemoryServer) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)